resources/language/resource.language.nl_nl/strings.po

```diff
@@ -173,6 +173,70 @@ msgctxt "#30041"
 msgid "Widevine CDM is required"
 msgstr "Widevine CDM is nodig"
 
+msgctxt "#30042"
+msgid "Using a SOCKS proxy requires the PySocks library (script.module.pysocks) installed."
+msgstr "Using a SOCKS proxy requires the PySocks library (script.module.pysocks) installed."
+
+msgctxt "#30043"
+msgid "Extracting Widevine CDM"
+msgstr "Widevine CDM wordt uitgepakt"
+
+msgctxt "#30044"
+msgid "Preparing downloaded image..."
+msgstr "Het recovery image wordt voorbereid..."
+
+msgctxt "#30045"
+msgid "Uncompressing image..."
+msgstr "Het recovery image wordt uitgepakt..."
+
+msgctxt "#30046"
+msgid "This may take several minutes."
+msgstr "Dit kan enkele minuten duren."
+
+msgctxt "#30047"
+msgid "[I]Please do not interrupt this process.[/I]"
+msgstr "[I]Graag dit proces niet onderbreken.[/I]"
+
+msgctxt "#30048"
+msgid "Extracting Widevine CDM from image..."
+msgstr "Het Widevine CDM wordt van het recovery image gehaald..."
```

Comment:
Better "De Widevine CDM wordt uit het recovery image gehaald...

Reply:
We gebruiken nergens een lidwoord, dus misschien beter zonder lidwoord?

Reply:
Oké, kan ook, maar als je een lidwoord gebruikt, moet het met de" want CDM staat voor module: https://www.welklidwoord.nl/module